src/ci/bin/testDockerDeadlock.sh

```diff
@@ -28,6 +28,21 @@ set -o errexit
 
 echo "docker-compose exit code was ${exit_code}"
 
+if test "${exit_code}" -gt "0"
```

Comment:
TOL since we're bashing: 

```
if [[ ${exit_code} -gt 0 ]]
```

Reply:
Huh, I thought `test xyz` was the new `[[ xyz ]]`... have I been misled? 😨